src/platforms/android/common/proguard.mdx

```diff
@@ -13,3 +13,13 @@ Support for Guardsquare's ProGuard and [DexGuard](https://www.guardsquare.com/de
 The Android SDK ships with ProGuard rules automatically defined; no further configuration is needed.
 
 </Note>
+
+## In-app Frames
+
+If you have an deobfuscated build, the Android SDK will automatically set all frames coming from the app as in-app, based on the `applicationId/packageName`. This is not possible on the client side for obfuscated builds. But you can define server-side rules for identifying the in-app frames, for example:
```

Comment:
Small wording tweak. Also we might need to hyphenate deobfuscate, but will circle back on that.
```suggestion
If you have an deobfuscated build, the Android SDK will automatically set all frames coming from the app as in-app, based on the `applicationId/packageName`. This is not possible on the client side for obfuscated builds. However, you can define server-side rules for identifying the in-app frames, for example:
```